setlexsem/generate/generate_sets.py

```diff
@@ -210,7 +211,7 @@ def make_sets(
     )
     all_sets = []
     for hp in make_hps_generator:
-        random_state = random.Random(seed_value)
+        # random_state = random.Random(seed_value)
```

Comment:
i'm worried about this, if we don't refresh the seed, the order of how hyperparameters are loaded can change the dataset we use.

It's okay for those cases that we read the dataset, but for those cases we/user wants to generate data on the fly, this affects things.

Reply:
I added it back in a new commit!